### PR TITLE
Resource manager: Native ansible collection

### PR DESCRIPTION
**Pull request by umarali-nagoor**
Unit Test Results

Screenshot 2023-09-01 at 1 04 55 PM

### CODE REVIEW

**Comment by umarali-nagoor** [comment]
Sanity Test Results

Screenshot 2023-09-01 at 1 04 46 PM

**Comment by umarali-nagoor** [comment]
Functionality Testing (playbook exection)


Screenshot 2023-09-01 at 12 51 45 PM
at 12.51.50 PM.png…]()

Screenshot 2023-09-01 at 12 51 58 PM

File: galaxy.yml [hunk 1]
```diff
@@ -1,11 +1,10 @@
 namespace: "ibm"
 name: "cloud"
-version: "0.0.1-beta1"
+version: "1.0.0"
 readme: "README.md"
 authors:
-    - "Mujeebur Rahman"
     - "Harini Kantareddy"
-    - "Kavya Handadi"
```
Comment (kavya498) [inline galaxy.yml:8]: @umarali-nagoor , Please revert the changes of the authors. Add your name along with the exiting authors.

Same applies to the authors in modules.

File: plugins/modules/ibm_resource_group.py [hunk 2]
```diff
@@ -13,8 +13,8 @@
 ---
 module: ibm_resource_group
 short_description: Manage C(resource_groups) for Resource Manager.
-author: Kavya Handadi (@kavya498)
-version_added: "0.0.1-beta0"
+author: Umar Ali (@umarali-nagoor)
+version_added: "1.0.0"
```
Comment (jaywcarman) [inline plugins/modules/ibm_resource_group.py:19]: The `version_added` value should remain at the version that the module was added. Same for the rest of the modules.

**Comment by jaywcarman** [comment]
I'm guessing this is probably already planned, but would be great to run tests in GH actions.

**Review by hkantare — APPROVED** [review]

File: plugins/modules/ibm_resource_group.py (hunk 2 above)
Comment (hkantare) [reply plugins/modules/ibm_resource_group.py:19]: @umarali-nagoor Can you check on this
